homebrew-proposal.rst

```diff
@@ -0,0 +1,28 @@
+The initial design by Jon and me is the following prose:
+
+Linking Travis-CI or Buildbot to the homebrew-tap repository would save:
+   #. The release engineer from logging into the Mac Mini and testing the recipe.
+   #. A reviewer of the PR to homebrew-tap’s master logging into the Mac mini and testing the recipe.
+
+It would not provide the other benefits of Continuous Integration.
+That is, CI should be able to tell you before you merge a commit into master whether that commit will break something.
```

Comment:
That depends on exactly what is being tested. It should be possible to generate a homebrew script targeting any particular commit, install that, and run tests against it.